frontend/public/module/k8s/types.ts

```diff
@@ -1094,3 +1094,77 @@ export type ConsolePluginKind = K8sResourceCommon & {
     };
   };
 };
+
+export type PDBCondition = {
+  observedGeneration?: number;
+} & K8sResourceCondition;
+
+export type PodDisruptionBudgetKind = {
+  spec: {
+    maxUnavailable?: number | string;
+    minAvailable?: number | string;
+    selector: Selector;
+  };
+  status?: {
+    conditions: PDBCondition[];
+    currentHealthy: number;
+    desiredHealthy: number;
+    disruptedPods?: Object;
+    disruptionsAllowed?: number;
+    expectedPods?: number;
+    observedGeneration?: number;
+  };
+} & K8sResourceCommon;
+
+export type K8sPodControllerKind = {
```

Comment:
Think we could give this guy a special block comment?

Something like:
```suggestion
/**
 * Not a real resource kind. A shared resource kind between resources that control pods.
 * eg. Deployment, ReplicaSet, etc..
 */
export type K8sPodControllerKind = {
```